networks/rpc/types_test.go

```diff
@@ -24,6 +24,8 @@ import (
 	"encoding/json"
 	"testing"
 
+	"github.com/klaytn/klaytn/common"
+
 	"github.com/klaytn/klaytn/common/math"
```

Comment:
There's an enter between two lines.
```suggestion
	"github.com/klaytn/klaytn/common"
	"github.com/klaytn/klaytn/common/math"
```

Reply:
I reflected it. 
Thank you yoomee1313!